src/BlazorWebView/src/Maui/Android/AndroidWebKitWebViewManager.cs

```diff
@@ -88,19 +88,6 @@ internal void SetUpMessageChannel()
 			_webview.PostWebMessage(new WebMessage("capturePort", destPort), AndroidAppOriginUri);
 		}
 
-		protected override async ValueTask DisposeAsyncCore()
-		{
-			await base.DisposeAsyncCore();
-
-			if (_nativeToJSPorts is not null)
```

Comment:
I don't believe we need the global `_nativeToJSPorts` field anymore, can we make that a local var again?